Umbraco-Cloud/Set-Up/Examine-Configuration/index.md

```diff
@@ -17,7 +18,21 @@ A provider is defined as
         supportProtected="true"
         analyzer="Lucene.Net.Analysis.WhitespaceAnalyzer, Lucene.Net"/>
 
-The above is the default internal indexer for an Umbraco installation. In order to specify where the indexes are stored we need to add an attribute called `useTempStorage` to the providers.
+The above is the default internal indexer for an Umbraco installation.
+
+#### Examine v0.1.80+ ####
```

Comment:
This needs to be completely removed. Cloud handles all of this by transforms, the customer doesn't need to do anything and in fact when working locally you should definitely not have a directoryFactory listed since it will Sync and store your indexes to strange locations.